src/jahongir/service/CategoryService.java

```diff
@@ -15,43 +15,58 @@ public Category[] getCategories() {
     @Override
     public boolean add(Object object) {
         Category category = (Category) object;
-
-        if (validate(category)) {}
-
-        return false;
+        if (validate(category)) {
+            categories[indexCategories++] = category;
+            return true;
+        } else {
```

Comment:
there is not needed "else" because anyway, it will pass "if" statement, if it will not enter to "if" statement.
(bu yoda else yozishish shartamas. chunki if shartiga tushmasa babir otib ketadi return false ni ozini yozb qoysezam boladi.